.docs/user-guide/automation.md

```diff
@@ -14,7 +14,32 @@ orchestration tools below.  Optionally, Docker is also listed in some examples.
 
 <br>
 ## Ansible
-ToDo
+With Ansible, installing the latest REX-Ray binaries can be accomplished by
+including the `codenrhoden.rexray` role from Ansible Galaxy.  The role accepts
+all the necessary variables to properly fill out your `config.yml` file.
```

Comment:
Well crap, I forgot I asked you to do that. It's that kind of day...

Yeah, that stuff can be added / modified later. Thanks!